.travis.yml

```diff
@@ -30,49 +30,6 @@ before_install:
           wget https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh -O $HOME/download/miniconda.sh;
         fi;
     fi;
-  # download test data for scatac
```

Comment:
I saw you had the test and then you removed the test for scatac. are the data too big for travis?

Reply:
Right. This image of travis has the time limit of 50 min and space limit of ~24GB. However, downloading the giggle index will exceed both the time and space limit.